index.html

```diff
@@ -1226,6 +1244,52 @@
       <a href="#steps-receiving">Receiving and parsing content</a>
       and <a>Writing or pushing content</a> sections.
     </p>
+    <p>
+      To <dfn>convert an NDEFRecord object's bytes</dfn> to a given <var>type</var>, run these steps:
+    </p>
+    <ol>
+      <li>Let <var>bytes</var> be the <a>NDEFRecord</a> object's associated <a>bytes</a>.
+      </li>
+      <li>Let <var>recordType</var> be the value of <a>NDEFRecord</a> object's recordType attribute.
+      </li>
+      <li>Switch on <var>type</var>:
+        <dl>
+          <dt>text</dt>
+          <ol>
+            <li>
+              If the <var>recordType</var> value is not equal to <code>"empty"</code>, then
+              return the result of running <a>UTF-8 decode</a> on the <var>bytes</var>.
+            </li>
+            <li>
+              Otherwise, return <code>null</code>.
+            </li>
+          </ol>
+          <dt>arrayBuffer</dt>
+          <ol>
+            <li>
+              If the <var>recordType</var> value is equal to <code>"json"</code> or
+              <code>"opaque"</code>, then return an {{ArrayBuffer}} whose contents are the
+              <var>bytes</var>. Exceptions thrown during the creation of the {{ArrayBuffer}}
+              object are re-thrown.
+            </li>
+            <li>
+              Otherwise, return <code>null</code>.
+            </li>
+          </ol>
+          <dt>JSON</dt>
+          <ol>
+            <li>
+              If the <var>recordType</var> value is equal to <code>"json"</code> or
+              <code>"opaque"</code>, then return the result of running <a>parse JSON from bytes</a>
+              on <var>bytes</var>. Re-throw any exceptions thrown by <a>parse JSON from bytes</a>.
```

Comment:
Here the prose for the rethrowing is different that above. Please be consistent

Reply:
Thanks, fixed.